src/components/modals/Username.js

```diff
@@ -44,17 +49,28 @@ class UsernameModal extends Component {
 
   isUsernameValid(username) {
     if (!username || username.length === 0) {
+      this.setState({
+        displayAlert: true,
+        failMessage: 'Must Not Be Empty'
```

Comment:
@MartyTheeMartian How about we decide whether to display the alert depending on if `!!failMessage` is true rather than having a separate variable?